fusion_power_video.cc

```diff
@@ -239,29 +245,24 @@ bool BrotliDecompress(const uint8_t* in, size_t size, size_t* pos,
   return true;
 }
 
-// Returns average entropy per symbol (a guess of bits per pixel).
+template<typename T> T approxLog2(T v) {
+   return ((unsigned) (8*sizeof(T) - __builtin_clzll((v)) - 1));
```

Comment:
__builtin_clzll is not available in all compilers, a fallback and guarding it with #if would be useful